webapp/data_structures/MAXMIN.js

```diff
@@ -13,7 +11,7 @@ function MAXMIN()
 
         var exprs = new ScilabString(...math.transpose([[2, 1, 1]]));
 
-        this.displayParameter = ["MAX"];
+        this.displayParameter = ["MIN/MAX"];
```

Comment:
```suggestion
        this.displayParameter = ["MAX"];
```
The earlier change was correct.